pkgs/os-specific/linux/libnl/default.nix

```diff
@@ -19,6 +19,8 @@ stdenv.mkDerivation {
       sha256 = "0dd7xxikib201i99k2if066hh7gwf2i4ffckrjplq6lr206jn00r";
     });
 
+  enableParallelBuilding = true;
```

Comment:
Not that this hurts anything - but I'm kind of amazed this would fix cross compiling. Usually parallel building breaks stuff. This is the first time I've heard of it fixing anything.

Reply:
It helps all the time when cross-compiling*!

* not fixing errors but making the rebuild/rebuild/rebuild go a bit faster and reduces the toll on your sanity!

Reply:
> not fixing errors but making the rebuild/rebuild/rebuild go a bit faster and reduces the toll on your sanity!

Couldn't have said it better myself.